scripts/stackstorm.js

```diff
@@ -283,6 +293,12 @@ module.exports = function(robot) {
       } else {
         data = req.body;
       }
+      // Special handler to try and figure out when a hipchat message
+      // is a whisper:
+      if (robot.adapterName == 'hipchat' && !data.whisper && data.channel.indexOf('@') > -1 ) {
```

Comment:
Realized this is actually a pretty bad test, I need to find a way to make this recognize a JID vs a room name, as room names can have @ symbols in them